pkg/apis/allocation/v1/gameserverallocation.go

```diff
@@ -77,6 +77,12 @@ type GameServerAllocationSpec struct {
 	// This is useful for things like smoke testing of new game servers.
 	Preferred []GameServerSelector `json:"preferred,omitempty"`
 
+	// Ordered list of GameServer label selectors.
+	// If the first selector is not matched, the selection attempts the second selector, and so on.
+	// This is useful for things like smoke testing of new game servers.
+	// Note: This field can only be set if neither Required or Preferred is set.
```

Comment:
I don't see a check in `Validate` that enforces this.

Reply:
Done.